pkg/security/admission/admission_test.go

```diff
@@ -293,18 +223,111 @@ func TestAdmit(t *testing.T) {
 		Level: "s0:c1,c0",
 	}
 
+	// level matches a value from namespace (see CreateNamespaceForTest())
+	seLinuxLevelFromNamespace := "s0:c1,c0"
```

Comment:
We can't pull this from the namespace?  I took a quick look but didn't see that it was available.  I guess the whole point is that if the ns method changes all of these tests are broken.  Easy to change since it's in one place.  If that's how it has to be then I'm ok with it.

Reply:
@pweil- I added what you've asked: https://github.com/openshift/origin/pull/17296/commits/268aea6fb58ed3e7a1e1d2f2f3a08f1050106fb0 PTAL. I don't see a way of how to do it better than that.

Reply:
you could make the `setupClientSet` call return the namespace that was used.  But yes, that's what I was looking to see so that if it is changed somewhere it won't break anything.  Thanks for the update.

Reply:
We need a namespace only in a single place, so I think it's better to keep this function as-is.